src/api/catalog.ts

```diff
@@ -125,11 +156,14 @@ export default ({config, db}) => async function (req, res, body) {
         let resultProcessor = factory.getAdapter(entityType, indexName, req, res)
 
         if (!resultProcessor) { resultProcessor = factory.getAdapter('default', indexName, req, res) } // get the default processor
-
         if (entityType === 'product') {
-          resultProcessor.process(_resBody.hits.hits, groupId).then((result) => {
+          resultProcessor.process(_resBody.hits.hits, groupId).then(async (result) => {
             _resBody.hits.hits = result
             _cacheStorageHandler(config, _resBody, reqHash, tagsArray)
+            if (_resBody.aggregations) {
```

Comment:
This way this feature won't work for graphql endpoint. graphql resolvers are using processors. we should finda way way to have it supported by graphql resolvers as well

Reply:
I've changed it a little bit and add graphql support https://github.com/DivanteLtd/vue-storefront-api/pull/399/files#diff-fe1dd7c73a325a992278d55bfa329092R72